src/alchemlyb/tests/parsing/test_namd.py

```diff
@@ -1,13 +1,25 @@
 """NAMD parser tests.
 
 """
```

Comment:
Please do not use `tempfile` and manual cleaning up. Instead use the `tmpdir` and `tmp_path` fixtures of pytest https://docs.pytest.org/en/6.2.x/tmpdir.html.